pkg/kubelet/dockertools/fake_docker_client.go

```diff
@@ -206,11 +212,26 @@ func (f *FakeDockerClient) StartContainer(id string, hostConfig *docker.HostConf
 			Config:     &docker.Config{Image: "testimage"},
 			HostConfig: hostConfig,
 			State: docker.State{
-				Running: true,
-				Pid:     os.Getpid(),
+				Running:   true,
+				Pid:       os.Getpid(),
+				StartedAt: time.Now(),
 			},
 			NetworkSettings: &docker.NetworkSettings{IPAddress: "1.2.3.4"},
 		}
+		if f.ContainerMap != nil {
```

Comment:
It's really not so bad... Sometimes it's worth pushing a little bit because people rarely come back to fix TODOs in the tests. 

After #14435 merges, https://github.com/kubernetes/kubernetes/commit/ba83f5dbf086dc9f86ccd16e95bb0face00fcbda is all you need to clean up the tests. It's almost entirely mechanical.